### ISSUE TITLE
obj vs no_obj always zero with latest yolo_layer.c

### ISSUE THREAD

**Issue by hellboy5** [AlexeyAB/darknet#7214]
Hi

I am using the latest commit and when I turn on show_details=1 and comment out ....

 //fprintf(stderr, "v3 (%s loss, Normalizer: (iou: %.2f, cls: %.2f) Region %d Avg (IOU: %f, GIOU: %f), Class: %f, Obj: %f, No Obj: %f, .5R: %f, .75R: %f, count: %d, class_loss = %f, iou_loss = %f, total_loss = %f \n",
        //    (l.iou_loss == MSE ? "mse" : (l.iou_loss == GIOU ? "giou" : "iou")), l.iou_normalizer, l.obj_normalizer, state.index, tot_iou / count, tot_giou / count, avg_cat / class_count, avg_obj / count, avg_anyobj / (l.w*l.h*l.n*l.batch), recall / count, recall75 / count, count,
        //    classification_loss, iou_loss, loss);

avg_obj and avg_anyobj fileds are always zero? It use to be before I came to the latest commit (using version from november) it was at least .5 in the beginning , now nothing? Any ideas? I know there have been some threads about using previous commit given all the changes to yolo_layer to support new_coords and scaled yolov4. 

Thanks
